code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -785,7 +791,7 @@
  */
 /mob/living/silicon/robot/proc/make_shell(obj/item/borg/upgrade/ai/board)
 	if(!board)
-		upgrades |= new /obj/item/borg/upgrade/ai(src)
+		src.upgrades |= new /obj/item/borg/upgrade/ai(src)
```

Comment:
Stop adding these random src.'s please

Reply:
upgrades is not accessible if src.upgrades is not explicitly declared in this case, , as far as I could tell from testing.